platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapGestureDetector.java

```diff
@@ -230,7 +230,7 @@ boolean onGenericMotionEvent(MotionEvent event) {
           float scrollDist = event.getAxisValue(MotionEvent.AXIS_VSCROLL);
 
           // Scale the map by the appropriate power of two factor
-          transform.zoomBy(Math.pow(2.0, scrollDist), event.getX(), event.getY());
+          transform.zoomBy(scrollDist, event.getX(), event.getY());
```

Comment:
Note that `Transform::zoomBy` used to be misleadingly named: it actually was a `scaleBy` operation. Now it's really `zoomBy`, and consequently we take the log2 of the input.